packages/core-bridge/index.d.ts

```diff
@@ -63,12 +63,13 @@ export interface ConsoleLogger {
  */
 export interface ForwardLogger {
   forward: {
+    // FIXME: This is no longer supported in core
     /**
      * What level, if any, logs should be forwarded from core at
      *
      */
     // These strings should match the log::LevelFilter enum in rust
-    level: LogLevel;
+    // level: LogLevel;
```

Comment:
FYI this can be supported the way it looks still, by just constructing an appropriate filter string.

Then we could deprecate it and add a filter string field.

Reply:
Definitely. TBH I'd personally go for what @cretz did in Python (only support level, not a filter string), but that might be a more breaking change.

Reply:
I think you flipped what you meant there, but yeah I agree. I suspect very few people use/care about this, so switching to filter string makes sense to me.

Reply:
The thing is that unless I'm missing something, the rust side now accept only a filter string, not tracingFilter+level. We can easily build a filter string from a level, but then, we would have to ignore `telemetryOptions.tracingFilter` in the forward case.

Reply:
Essentially, we now need a filter string on the logger (either Console or Forward) and a filter on the trace exporter.

Reply:
If had to design an API for this at this moment, _without considering backward compatibility_, it would looks something like this:

```
export interface TelemetryOptions {
  logging: {
    filter: string,
    mode: 'console' | 'forward'
  },

  tracing: {
    filter: string;
    ...
  }
}

function makeFilterString(sdkLevel: Level, otherLevel: Level = 'WARN') { ... }
```

That is roughly how [Python defines this](https://github.com/temporalio/sdk-python/pull/191/files#diff-0d990372bfa478206aff7cbb03297d8f4423f41304104e26474d159eef870c65R27-R43).

Reply:
I agree, it's better than what we have now.
I wish I'd used `mode` instead of having the nested `console` and `forward` objects.

Reply:
But given the current API, I would propose this instead: 

```
export interface TelemetryOptions {
  /** @deprecated */
  tracingFilter?: string

  logging: {
    // If undefined, then default to these values, in that order:
    //   1. makeFilterString(forward.level, forward.level)  [if forward.level exists]
    //   2. TelemetryOptions.tracingFilter  [if it exists]
    //   3. makeFilterString('INFO', 'INFO')  [if everything else is undefined]
    filter?: string,
    console: {}    // Either 'console' or 'forward', not both
    forward: {
      /** @deprecated */
      level: Level;
    }
  },

  tracing: {
    // If undefined, then default to these values, in that order:
    //   1. TelemetryOptions.tracingFilter  [if it exists]
    //   2. makeFilterString('INFO', 'INFO')  [if everything else is undefined]
    filter?: string;
    ...
  }
}

function makeFilterString(sdkLevel: Level, otherLevel: Level = 'WARN') { ... }
```

Reply:
It's complicated though. Depends on how far we need to go in backward compatibility...

Reply:
And in any cases, these backward compatibility things would be handled in TS. Bridge would accept a `CompiledTelemetryOptions`, expecting something more in line with my first proposition.